systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -204,7 +113,7 @@ protected void changeKafkaAndLogFormatVersion(JsonObject procedures, JsonObject
         String toUrl = testParameters.getString("urlTo");
         String examplesPath = "";
         if (toUrl.equals("HEAD")) {
-            examplesPath = io.strimzi.systemtest.Constants.PATH_TO_PACKAGING_EXAMPLES + "";
+            examplesPath = PATH_TO_PACKAGING_EXAMPLES + "";
```

Comment:
```suggestion
            examplesPath = Constants.PATH_TO_PACKAGING_EXAMPLES + "";
```

Reply:
There is a problem with 
import io.strimzi.api.kafka.model.Constants;
and 
io.strimzi.systemtest.Constants.*; 
seems like I will have to revert this change of mine.